querydsl-jpa/src/main/java/com/querydsl/jpa/impl/JPAProvider.java

```diff
@@ -39,9 +46,23 @@ private static void addMapping(String className, JPQLTemplates templates) {
     }
 
     static {
+        boolean hibernate5;
+
+        try {
+            String version = Class.forName("org.hibernate.Session").getPackage().getImplementationVersion();
+            String[] versionParts = version.split("\\.");
+            int major = Integer.parseInt(versionParts[0]);
+//            hibernate5 = major >= 5;
```

Comment:
Should be fixed, but I want to see how this affects the test suite. So I will enable this in a separate commit.